bag_verifier/src/test/scala/weco/storage_service/bag_verifier/services/BagVerifierTestCases.scala

```diff
@@ -726,34 +730,6 @@ trait BagVerifierTestCases[Verifier <: BagVerifier[
           )
       }
     }
-
-    it("passes a bag that includes an extra manifest/tag manifest") {
```

Comment:
Why did we have this test in the first place? Seems an unusual thing to worry about.

Reply:
Somewhere in the bag verifier there's a line that says _"look for any files in the bag that are (1) listed in one of the manifests or (2) the SHA-256 manifest"_.

Goobi uploads both SHA-256 and SHA-512 tag manifests, but until this BagReader patch we had to explicitly ignore for the extra tag manifest. We had to test the storage service wouldn't complain about a tag manifest it wasn't reading. I'm going to rework this a bit more in the verifier patch – it was easier to remove this test than get it working.